phoenix-core/src/it/java/org/apache/phoenix/end2end/index/DropColumnIT.java

```diff
@@ -522,7 +523,8 @@ public void helpTestDroppingIndexedColDropsViewIndex(boolean isMultiTenant) thro
             byte[] cq = column.getColumnQualifierBytes();
             // there should be a single row belonging to VIEWINDEX2 
             assertNotNull(viewIndex2 + " row is missing", result.getValue(QueryConstants.DEFAULT_COLUMN_FAMILY_BYTES, cq));
-            assertNull(results.next());
+            // TODO enable this after we drop view indexes than need a dropped column 
+//            assertNull(results.next());
```

Comment:
what about this TODO?

Reply:
This test now passes without commenting the assertNull.